### ISSUE TITLE
Failed to build for Quicklisp dist

### ISSUE THREAD

**Issue by quicklisp** [cosmos72/hyperluminal-mem#9]
Building with SBCL 2.0.9.5-442f54894 / ASDF 3.3.1 for quicklisp dist creation.

Commit id 7dea5a4b082c13c1c2463341fa43d47249058f27

*hyperluminal-mem-test* fails to build because of a failure in  _hyperluminal-mem_.

*hyperluminal-mem* fails to build with the following error:

```
; caught ERROR:
;   READ error during COMPILE-FILE: Lock on package SB-VM violated when interning EAX-OFFSET while in package HYPERLUMINAL-MEM-SBCL.
;   See also:
;     The SBCL Manual, Node "Package Locks"(in form starting at line: 427, column: 0, position: 16785)
...
Unhandled UIOP/LISP-BUILD:COMPILE-FILE-ERROR in thread #<SB-THREAD:THREAD "main thread" RUNNING {100E838223}>: COMPILE-FILE-ERROR while compiling #<CL-SOURCE-FILE "hyperluminal-mem" "sbcl" "x86">

```

[Full log here](http://report.quicklisp.org/2020-10-06/failure-report/hyperluminal-mem.html)

**Comment by cosmos72** [comment]
Thanks for spotting this!

fixed in commit e3437d77b774069e0aeaa52e52cd083b926076b4 - it's the same problem as https://github.com/cosmos72/stmx/issues/20 :
on x86-64, sbcl 2.0.9 renamed some internal symbols needed to define VOPs:
````
sb-vm::eax-offset -> sb-vm::rax-offset
sb-vm::ebx-offset -> sb-vm::rbx-offset
sb-vm::ecx-offset -> sb-vm::rcx-offset
...
```

**Comment by quicklisp** [comment]
Thanks!
